aws-ts-ruby-on-rails/README.md

```diff
@@ -1,6 +1,6 @@
 [![Deploy](https://get.pulumi.com/new/button.svg)](https://app.pulumi.com/new)
 
-# AWS EC2 Ruby on Rails
+# Ruby on Rails Server Using Amazon EC2
```

Comment:
Why "**on** AWS Fargate" but "**Using** Amazon EC2"? I'm sure you have a good reason but I don't know it yet :)

Reply:
I think I just got lazy. :) Let's change to "Using" on the Fargate one.